resources/views/sprint/view.blade.php

```diff
@@ -29,16 +29,15 @@
 			</ul>
 
 			@if(Auth::check() && Auth::user()->isInAdminList(env('PHRAGILE_ADMINS')))
-				{!! link_to_route(
-					'delete_sprint_path',
-					'',
-					['sprint' => $sprint->phabricator_id],
-					[
-						'class' => 'btn btn-danger btn-lg glyphicon glyphicon-remove',
-						'title' => 'Delete sprint',
-						'onclick' => 'return confirm("Delete this sprint?")'
-					]
-				) !!}
+				{!! Form::open(['route' => ['delete_sprint_path', $sprint->phabricator_id], 'method' => 'DELETE']) !!}
+					{!! Form::submit(
+						'&#xe014;', [
```

Comment:
When using the Glyphicons Halflings that prints the "remove" symbol. Previously this has been done by bootstrap css (styling `:before` of links) but this cannot be done the same way for submit buttons.
Your comment reminded me that I forgot that it makes no use to add `glyphicon-remove` class to new delete buttons, thanks!